utils/pretty_printers.cc

```diff
@@ -11,16 +11,22 @@
 namespace utils {
 
 std::ostream& operator<<(std::ostream& os, pretty_printed_data_size data) {
-    static constexpr const char *suffixes[] = {" bytes", "kB", "MB", "GB", "TB", "PB"};
-
-    unsigned exp = 0;
-    while ((data._size >= 1000) && (exp < sizeof(suffixes))) {
-        exp++;
-        data._size /= 1000;
+    static constexpr char* suffixes[] = {" bytes", "kB", "MB", "GB", "TB", "PB"};
```

Comment:
The compiler caught a real bug here - you forgot the "const".
Please just don't modify the original line we had here, it was correct.

Reply:
sorry, i don't have access to my building machine at this moment. sure, reverted.